assets/textures/paper.docx

```diff
@@ -0,0 +1,8 @@
+# struct subtexture
+# one sprite, as part of a texture atlas.
+# 
+# this struct stores information about positions and sizes
+# of sprites included in the 'big texture'.
```

Comment:
Copy-pasted from other generated docx. Probably, only the 'x,y,w,h,cx,cy' comment line is useful.